web/src/components/tasks/NaicsCodeDisplay.tsx

```diff
@@ -21,19 +24,32 @@ export const NaicsCodeDisplay = (props: Props): ReactElement => {
       <Alert variant="success">
         <Content>{templateEval(Config.determineNaicsCode.successMessage, { code: props.code })}</Content>
       </Alert>
-      <div className="margin-left-3">
+      <div className="margin-left-3 flex">
         <span className="text-bold margin-right-05">{props.code}</span>
         {"-"}
         <span className="margin-left-05 margin-right-2">
           {lookupNaicsCode(props.code)?.SixDigitDescription}
         </span>
-        <UnStyledButton style="tertiary" underline onClick={props.onEdit}>
-          {Config.taskDefaults.editText}
-        </UnStyledButton>
-        <span className="margin-x-105">|</span>
-        <UnStyledButton style="tertiary" underline onClick={props.onRemove}>
-          {Config.taskDefaults.removeText}
-        </UnStyledButton>
+        {props.lockField ? (
+          <ArrowTooltip
+            title={Config.profileDefaults.lockedFieldTooltipText}
+            data-testid="naics-code-tooltip"
+          >
+            <div className="fdr fac font-body-lg">
+              <Icon>help_outline</Icon>
+            </div>
+          </ArrowTooltip>
```

Comment:
As a note (maybe we should add this as a refactoring chore)

After Faraz merges his changes to the Business STructure task, he made some almost identical changes to lock the business structure alert versus showing the edit/remove text there.

After both these are merged, we should refactor these to use a shared component for the shared behavior